apps/mosaico/src/Mosaico/Frontpage/Events.purs

```diff
@@ -0,0 +1,29 @@
+module Mosaico.Frontpage.Events where
+
+import Prelude
+
+import Data.Maybe (Maybe(..), maybe)
+import Data.Nullable (toMaybe)
+import Effect (Effect)
+import React.Basic.Events (EventHandler)
+import React.Basic.DOM.Events (capture, target)
+import Unsafe.Coerce (unsafeCoerce)
+import Web.DOM (Node)
+import Web.DOM.Node as Node
+
+onFrontpageClick :: (String -> Effect Unit) -> EventHandler
+onFrontpageClick setRoute = capture target $ \tgt -> do
+  let findAnchor :: Node -> Effect (Maybe String)
+      findAnchor node = do
+        let continue = maybe (pure Nothing) findAnchor =<< Node.parentNode node
+        case Node.nodeName node of
+          "A" ->
+            case toMaybe $ (unsafeCoerce node).attributes.getNamedItem "href" of
```

Comment:
If there is a safe (and not much more complicated) way of reading the "href", why not do it that way?
We only seem to need `Web.DOM.Element.fromNode` and `Web.DOM.Element.getAttribute`.

Reply:
If there is, I couldn't find it. In https://pursuit.purescript.org/packages/purescript-web-dom/5.0.0/docs/Web.DOM.Node the PureScript types go only so far as to get a `Node` but there's no functions for accessing its attributes.

Reply:
On second look, yes, `Web.DOM.Element`. Could as well do it with functions in it.